docs/en/04_Changelogs/3.2.0.md

```diff
@@ -17,6 +17,7 @@
  * `Mailer` no longer calls `xml2raw` on all email subject line, and now must be passed in via plain text.
  * `ErrorControlChain` now supports reload on exceptions
  * `FormField::validate` now requires an instance of `Validator`
+ * Admin URL can now be configured via custom Director routing rule
```

Comment:
This is modifying the wrong changelog - your PR is against `master`, which will become the `4.0` release. Given this is a relatively low impact feature (no API change), I'd be happy for you to rebase this work against the `3` branch though (which will be released as 3.2 soon).